src/lib/Group.ts

```diff
@@ -110,24 +111,20 @@ export class Group<
    */
   async messages(
     skipSync: boolean = false,
```

Comment:
Want to include skipSync in the options? You can do something like:
```
interface GroupMessagesOptions extends MessagesOptions {
skipSync?: boolean
}
```
And you can remove some duplicate code doing:
```
const {
skipSync = false
} = opts ?? {}
```

Reply:
I'd be curious @cameronvoell thoughts here. He added skipSync all over the code base so I kind of felt like it belonged outside of the messageOptions as it's more of a conscious action. But I'm totally fine with moving it into the options if we're cool with that.

Reply:
For consistency with other functions I think it's slightly better to keep it as a parameter the way it is. 

Right now feels like the optional skipSync is the least bad way I've heard for having developers be generally aware that libxmtp group functions only return results from local data, and require a sync first if you want latest from network. Open to other suggestions for sure. But if we do keep it, I think we might as well have it passed as a parameter in a consistent way across functions.